pysb/simulator/cupsoda.py

```diff
@@ -23,6 +23,10 @@
     import pycuda.driver as cuda
 except ImportError:
     cuda = None
+try:
+    from collections.abc import Iterable
+except ImportError:
+    from collections import Iterable
```

Comment:
One more instance to remove the try/except block from. Thanks!

Reply:
Thanks for the catch, removed.